README.md

```diff
@@ -362,6 +363,14 @@ _Mailing lists servers and mass mailing software - one message to many recipient
 - [Roundcube](https://roundcube.net) - Browser-based IMAP client with an application-like user interface. ([Source Code](https://github.com/roundcube/roundcubemail)) `GPL-3.0` `PHP`
 - [SquirrelMail](http://squirrelmail.org) - Another browser-based IMAP client. ([Source Code](https://sourceforge.net/p/squirrelmail/code/HEAD/tree/)) `GPL-2.0` `PHP`
 
+### Usenet
```

Comment:
Please add this to Misc/Other

"Any category you are creating has the minimum requirement of 3 items. If not, your addition may be inserted into Misc/Other."